docs/DeveloperGuide.md

```diff
@@ -595,6 +601,47 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
   - 4a1. SalesPunch alerts the user that the lead status is the same.  
     Use case resumes at step 2
 
+**Use case: Add tasks for a person**
+
+**MSS**
+
+1.  User adds task to a person
+2.  SalesPunch updates the tasklist of the person in the contact list
+
+    Use case ends.
+
+**Extensions**
+
+- 1a. User is able to add more types of tasks to a person. 
+
+    - 1a1. SalesPunch requests for the correct data.
+    - 1a2. User enters new data.
+    - Steps 1a1-1a2 are repeated until the data entered is correct.
+    - Use case resumes from step 2.
+
+  Use case ends.
+
+**Use case: Edit tasks**
```

Comment:
This feature does not exist in our application. Consider removing this from the DG, or move this to Proposed Implementations, but be sure to add in more context and details about the implementation.